pkg/handlers/v1/attribute_test.go

```diff
@@ -0,0 +1,91 @@
+package v1
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+
+	"github.com/asecurityteam/nexpose-asset-attributor/pkg/assetattributor"
+	"github.com/asecurityteam/nexpose-asset-attributor/pkg/domain"
+	"github.com/asecurityteam/nexpose-asset-attributor/pkg/nexpose"
+	"github.com/asecurityteam/runhttp"
+	"github.com/golang/mock/gomock"
+
+	"github.com/asecurityteam/logevent"
+)
+
+func TestSuccess(t *testing.T) {
+	input := domain.NexposeAssetVulnerabilities{
+		Asset: &nexpose.Asset{
+			ID: 123,
+		},
+	}
+
+	handler := &AttributeHandler{
+		AssetAttributor: assetattributor.NewNoOpAssetAttributor(),
+		LogFn:           runhttp.LoggerFromContext,
```

Comment:
I recommend passing in a function that implements the expected signature that returns a mock implementation of the logger or stat client. Otherwise you're beholden to the various behaviors of the default implementations of those functions from `runhttp` in cases where the context is empty and which may change over time. Ex:

```golang
LogFn: func(context.Context) domain.Logger {
  return mockLogger
},
//...
mockLogger.EXPECT().Info(gomock.Any()).AnyTimes()
```